### ISSUE TITLE
Add NVDA bot to "Assign Testers" menu in test queue for NVDA test runs

### ISSUE THREAD

**Issue by mcking65** [w3c/aria-at-app#874]
Because of #873, I could not ask NVDA bot to run a draft plan with Firefox. We probably discussed the option of assigning to NVDA Bot, and we may have decided to not include it in the MVP. Now, especially after experiencing what NVDA Bot produces, it does seem like this could be a useful feature to add. That said, fixing #873 should be higher priority.

**Comment by jugglinmike** [comment]
@mcking65 I understand your request for Firefox support above, so I've filed gh-914 to track that specifically. I'm not certain this completely covers what you were looking for, though, so can you let me know if there any other enhancements that you had in mind?

**Comment by jugglinmike** [comment]
Matt confirmed that gh-914 does in fact supersede this.
